xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-python/pom.xml

```diff
@@ -60,4 +63,16 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <!-- Skip pit/descartes for now as it's failing on this module, see: https://github.com/STAMP-project/pitest-descartes/issues/72 -->
+      <plugin>
+      <groupId>org.pitest</groupId>
+      <artifactId>pitest-maven</artifactId>
+      <configuration>
+        <skip>true</skip>
```

Comment:
Do we need this even with `<xwiki.pitest.skip>true</xwiki.pitest.skip>` above?

Also shouldn't this be in the quality profile only?

Reply:
My bad, I forgot to remove the property. The property is not working as we override it by passing the argument in command line when executing pitest.

Reply:
> Also shouldn't this be in the quality profile only?

Not sure: actually we should always avoid launching pit/descartes on this module as it's failing, no matter if it's because we are in quality profile or for another reason.

Reply:
Except that right now, all the pitest/descartes definitions/configurations are only present in the quality profile in the top level pom.